src/api/app/controllers/trigger_controller.rb

```diff
@@ -33,12 +31,17 @@ def release
     raise NoPermissionForPackage.setup('not_found', 404, "#{@pkg.project} has no release targets that are triggered manually") unless manual_release_targets.any?
 
     manual_release_targets.each do |release_target|
+      raise NoPermissionForPackage.setup('not_permission', 403, "#{release_target.target_repository.project.name} project is not writable for user") \
```

Comment:
I changed this from `NoPermissionForProject` because I assumed it was a typo. If its not a typo then we'll have to create the new error type